ecs.tf

```diff
@@ -93,6 +101,25 @@ resource "aws_s3_bucket_object" "prometheus_config" {
   kms_key_id = data.terraform_remote_state.management.outputs.config_bucket.cmk_arn
 }
 
+resource "aws_service_discovery_private_dns_namespace" "prometheus" {
+  name = "services.${var.parent_domain_name}"
+  vpc  = module.vpc.outputs.vpc_ids[0]
+}
+
+resource "aws_service_discovery_service" "prometheus" {
```

Comment:
Need to take a look at this and understand where it's going to write DNS records and how they will resolve, as it might need to go into the management account

Reply:
Just a query it is a private DNS entry, so only available in the VPC. Not sure it's possible to write in a different account. Even if it is, would we still want to as it is not public?